contributors/guide/pull-requests.md

```diff
@@ -590,6 +590,26 @@ at once to that file.
 * Can the file be improved further?
 * Does the trivial edit greatly improve the quality of the content?
 
+## Fixing linter issues
+
+Kubernetes has a set of linter checks. Some of those must pass in the entire
+code base, some must pass in new or modified code, and some are merely hints
+to developers how to improve their code.
+
+Please do not create Pull Requests for issues found by linters without first
+reaching out to maintainers to determine whether there is sufficient interest
+in fixing such issues. It does not matter whether the linter is enabled in
+Kubernetes or not:
+- If a linter *is* enabled in
+  [golangci.yaml](https://github.com/kubernetes/kubernetes/blob/master/hack/golangci.yaml),
+  then it has already been determined that sweeping changes in the existing
```

Comment:
...aren't necessary or just are not worth the cost (e.g. causing rebases or obscuring authorship).

Reply:
Added, with one small change: "causing rebases" -> "causing rebases of other Pull Requests"